README.md

```diff
@@ -181,6 +181,13 @@ Finally, you may choose to overwrite some configuration defaults.
 
   > In lieu of a user session, a helper instance _could_ store the authorization state in a local variable and share it between users of the instance. Currently, this is not supported, because the primary audience for the library is thought to be a typical web application with unique user accounts.
 
+* `sessionAuthorizationKeySecret`
+
+  The helper aims to controlle the authorizationKey generation, which represents the key within the session object for the authorization state. A random authorizationKey is generated, if `'sessionAuthorizationKeySecret'` is not set, 
+  
+  > In order to scale an application instance horizontaly, while using a shared session store, the authorizationKey can be controlled so that an authorization state is available thoughtout parallel running instances. If `'sessionAuthorizationKeySecret'` is not set, a random authorizationKey is generated per application instance. 
```

Comment:
Same as the above, maybe just ticks, and there also seem to be a couple of typos: "horizontaly", "thoughtout". So, maybe something like this:
> In order to scale an application horizontally, while using a shared session store, the key, under which the authorization state is stored, can be controlled, so that an authorization is available throughout instances of the application running in parallel. If `authorizationId` is not set, a random key is generated for each helper instance.